cpp/src/examples/gcylon/gjoin.cpp

```diff
@@ -0,0 +1,84 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/table/table.hpp>
+#include <cudf/table/table_view.hpp>
+
+#include <gcylon/utils/construct.hpp>
+#include <gcylon/gtable_api.hpp>
+#include <cylon/net/mpi/mpi_communicator.hpp>
+#include <examples/gcylon/print.hpp>
+
+using std::cout;
+using std::endl;
+using std::string;
+using namespace gcylon;
+
+int main(int argc, char *argv[]) {
+
+    const int COLS = 2;
+    const int64_t ROWS = 10;
+
+    auto mpi_config = std::make_shared<cylon::net::MPIConfig>();
+    auto ctx = cylon::CylonContext::InitDistributed(mpi_config);
+    int myRank = ctx->GetRank();
```

Comment:
we should follow a consistant naming, this is a camel case naming while the other variable is with _